lib/netuitive/error_tracker.rb

```diff
@@ -0,0 +1,54 @@
+require 'netuitive/controller_utils'
+require 'netuitive/rails_config_manager'
+module ErrorTrackerHook
+  extend ActiveSupport::Concern
+
+  include ControllerUtils
+
+  def isIgnoredError(exception)
+    if !ConfigManager.ignoredErrors.empty?
```

Comment:
`return unless ConfigManager.ignoredErrors.empty?` and then you can unnest the rest a level